tools/dockerbuild.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+mkdir build
+cd build
+cmake ..
+make
+cd ..
+mkdir out
```

Comment:
```suggestion
mkdir -p build
cd build
cmake ..
make clean
make -j
cd ..
mkdir -p out
```

It will build way faster if multicore is available and won't break on rebuilds in MacOS

Reply:
good suggestion, thanks!